docs/devs/lib-test-specifications.md

```diff
@@ -39,23 +56,23 @@ curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.3/install.sh | bash
 # Download and install Node.js:
 nvm install 22
 
-# Verify the Node.js version:
+# Verify installation
 node -v # Should print "v22.20.0".
-
-# Verify npm version:
 npm -v # Should print "10.9.3".
 ```
 
-### Troubleshooting
-* Old versions of nodejs (v12) will give errors like `SyntaxError: Unexpected token '{'`.
+## TypeScript
+![Supported Versions](https://img.shields.io/badge/Supported_Versions-v5\.6\.3_--_v5\.8\.3-blue)
```

Comment:
Thanks for testing the various combinations. Is it worth leaving a footnote here about latest ts failing the tests? If I remember correctly, they still run correctly, but they print warnings which the test detects and decides its a failure (no warnings are allowed).

Reply:
Nothing wrong with the generated code, tests are all happy and generated TS code looks fine to me.

But yeah was more or less a typing/version issue. Having cleared up the pinned `@types/node`, we can now mix-match versions pretty much as we wish, they all seem to work equally well